src/utils/auth-channel.js

```diff
@@ -0,0 +1,43 @@
+import uuid from "uuid/v4";
+
+export default class AuthChannel {
+  constructor(store) {
+    this.store = store;
+    this.socket = null;
+  }
+
+  setSocket = socket => {
+    this.socket = socket;
+  };
+
+  removeCredentials = () => {
+    return this.store.update({ credentials: { token: "", email: "" } });
```

Comment:
reason for `""` instead of `null`?

Reply:
Hmm, I think the store schema validation didn't like nulls, but I'll check again.